br/pkg/lightning/backend/external/writer.go

```diff
@@ -488,6 +488,9 @@ func (w *Writer) createStorageWriter(ctx context.Context) (
 
 // EngineWriter implements backend.EngineWriter interface.
```

Comment:
maybe mention it's concurrent safe in comments

Reply:
https://github.com/pingcap/tidb/pull/47185/commits/58958f8e37d8ce18f58da11fbc6181bf3aa0a67e